### ISSUE TITLE
Folder Structure Inaccurate

### ISSUE THREAD

**Issue by jasonsansone** [HaveAGitGat/Tdarr#326]
**Describe the bug**
If ./Tdarr_Updater is run in a folder depth of only one deep, the folder structure [is as expected](https://tdarr.io/docs/).  For example, download and run ./Tdarr_Updater inside of /opt.  In such scenario, the /opt/configs will contain all appropriate json.  However, the folder structure design breaks down if a deeper root is utilized for Tdarr.  For example, install to /opt/tdarr.  The initial Tdarr_Updater_Config.json will be located at /opt/tdarr/configs.  However, subsequently running /opt/tdarr/Tdarr_Node/Tdarr_Node will place Tdarr_Node_Config.json at /opt/configs and not in the anticipated location of /opt/tdarr/configs.  The result is two different locations for files with potential permissions issues.

The following files and folders are effected:
configs
logs
server

**To Reproduce**
Install v2 on linux.  Test different depths of folder configurations.

OS: Ubuntu 20.04.1 LTS as LXC container on Proxmox 5.4.78-2
Tdarr Version: 2.00.00

**Comment by wsegatto** [comment]
Yep, mentioned in #317 and #327.

**Comment by HaveAGitGat** [comment]
The folder structure bug when running modules from a different dir should be fixed in 2.00.01. I've made some changes to Tdarr_Updater - the new version allows downloading of a new package without it being 'latest'.  So if you download the new updater (http://tdarr.io/docs/), and specify 2.00.01 in the config, it will grab that package. If you leave it as 'latest' it will still grab 2.00.00. Once fixes have been verified, I can set 2.00.01 as 'latest'.

```
{
  "platform_arch": "default",
  "modules": {
    "Tdarr_Desktop": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": "2.00.01"
    },
    "Tdarr_Node": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    },
    "Tdarr_Server": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    },
    "Tdarr_WebUI": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    }
  }
}
```

**Comment by jasonsansone** [comment]
> The folder structure bug when running modules from a different dir should be fixed in 2.00.01. I've made some changes to Tdarr_Updater - the new version allows downloading of a new package without it being 'latest'. So if you download the new updater (http://tdarr.io/docs/), and specify 2.00.01 in the config, it will grab that package. If you leave it as 'latest' it will still grab 2.00.00. Once fixes have been verified, I can set 2.00.01 as 'latest'.
> 
> ```
> {
>   "platform_arch": "default",
>   "modules": {
>     "Tdarr_Desktop": {
>       "enabled": true,
>       "requiredVersion": "2.00.01",
>       "currentVersion": "2.00.01"
>     },
>     "Tdarr_Node": {
>       "enabled": true,
>       "requiredVersion": "2.00.01",
>       "currentVersion": ""
>     },
>     "Tdarr_Server": {
>       "enabled": true,
>       "requiredVersion": "2.00.01",
>       "currentVersion": ""
>     },
>     "Tdarr_WebUI": {
>       "enabled": true,
>       "requiredVersion": "2.00.01",
>       "currentVersion": ""
>     }
>   }
> }
> ```

Tested working on macOS and Linux.  Closing as fixed.
